README.md

```diff
@@ -1,18 +1,29 @@
 # Common Utils
 A collection of common utility methods used by Intrepid.
 
+# Table of Contents
+1. [Setup](#setup)
+2. [Release](#release)
+3. [License](#license)
 
-Setup
-=======
+# Setup
 Add the following lines to your build.gradle file:
 ```
 dependencies {
     compile "io.intrepid.commonutils:commonutils:0.2.2"
 }
 ```
 
-License
-=======
+# Release
+To release a new version of this library, follow these steps:
+1. Create a new branch containing the version # in its name
+2. Increment the `versionName` in `build.gradle` (changing the `versionCode` is not required)
+3. Update the current version number in `README.md`
+4. Add a new entry for the current version in `CHANGELOG.md`
+5. Submit a PR against the `develop` branch with your changes
+6. Once approved, squash and merge your changes and then ask the `#android` Slack channel about how to publish the new version to Bintray.
```

Comment:
We should include the actual publishing instructions here (setting the local properties and running `./gradlew bintrayUpload`). 
```
bintray_user=####
bintray_key=####
sonatype_user=####
sonatype_password=####
```
Don't reference the android slack channel since this is a public project and the instructions should be applicable to everyone.